### PR TITLE
Updated TUIChip

### PR DESCRIPTION
**Pull request by maheshwaran01m**
- Replaced with style

### CODE REVIEW

**Comment by arvindhsukumar** [comment]
@maheshwaran01m Does this need to go into the FloatingActionButton branch, or can it go into develop directly?

**Comment by maheshwaran01m** [comment]
@arvindhsukumar I need some things in that branch, like `border` modifiers

**Comment by arvindhsukumar** [comment]
> @arvindhsukumar I need some things in that branch, like `border` modifiers

Ok, but please take care to keep PRs separate going forward

**Review by arvindhsukumar — APPROVED** [review]

**Comment by maheshwaran01m** [comment]
@arvindhsukumar Sure

File: Sources/TarkaUI/Components/Base Components/Chip/Modifiers/TUIChip+Modifiers.swift [hunk 1]
```diff
@@ -14,10 +14,10 @@ public extension TUIChip {
   ///   - style: Style can be assist, input, suggestion, Filter
   ///   - size: size32, size40
   /// - Returns: A closure that returns the TUIChip
-  func style(_ style: Style, size: Size = .size40) -> Self {
+  func style(_ style: ChipStyle, size: Size = .size40) -> Self {
     var newView = self
-    newView.style = style
-    newView.size = size
+    newView.style.chipStyle = style
+    newView.style.size = size
```
Comment (gopinathcholan) [inline Sources/TarkaUI/Components/Base Components/Chip/Modifiers/TUIChip+Modifiers.swift:20]: what is this style.chipStyle ?

File: Sources/TarkaUI/Components/Base Components/Chip/Modifiers/TUIChip+Modifiers.swift [hunk 2]
```diff
@@ -27,47 +27,78 @@ public extension TUIChip {
...
+    newView.style.isSelected = isSelected
+    newView.style.badgeCount = badgeCount
+    newView.style.badgeColor = badgeColor
     return newView
   }
   
   func action(_ action: @escaping () -> Void) -> Self {
     var newView = self
-    newView.action = action
+    newView.style.action = action
```
Comment (gopinathcholan) [inline Sources/TarkaUI/Components/Base Components/Chip/Modifiers/TUIChip+Modifiers.swift:41]: how can the style holds action?

Comment (maheshwaran01m) [reply Sources/TarkaUI/Components/Base Components/Chip/Modifiers/TUIChip+Modifiers.swift:41]: @gopinathcholan we are using many values, so I have created struct

Comment (maheshwaran01m) [reply Sources/TarkaUI/Components/Base Components/Chip/Modifiers/TUIChip+Modifiers.swift]: @gopinathcholan Updated with `inputItem`

**Review by gopinathcholan — APPROVED** [review]

**Pull request merged**